html/changelogs/burgerbb-biogenerator.yml

```diff
@@ -0,0 +1,12 @@
+author: BurgerBB
+delete-after: True
+changes: 
+  - tweak: "Improved code and UI."
```

Comment:
This, and the report below it should be rolled into one. And given context. Right now it's unclear as to _what_ had it's code and UI improved.